data_structures/skip_list.cpp

```diff
@@ -0,0 +1,207 @@
+/**
+ * A skip list is a data structure that is used for storing a sorted list of items with a
+ *  help of hierarchy of linked lists that connect increasingly sparse subsequences of the items
+ * 
+ *  References used: GeeksForGeeks skip list code, https://iq.opengenus.org/skip-list/ Pseudo Code.
+*/
+
+#include <iostream>
+#include <vector>
+#include <string.h>
+#include <stdlib.h>
+#include <ctime>   
+
+// using namespace std; 
+using std::vector;
+using std::endl;
+
+#define MAXLVL 2   ///< maximum level of skip list
+#define P 0.5      ///< current probability for "coin toss"
+  
+
+/**
+ *  Node structure [Key][Node*, Node*...]
+*/
+struct Node { 
+    int key; 
+    void* value;
+    /*Forward Array*/
+    vector<Node*> forward;
```

Comment:
missing class member documentations

Reply:
done

Reply:
Please review the documentation for classes. This is not per the doxygen standards as shared in the links.